code/controllers/subsystem/garbage.dm

```diff
@@ -46,116 +50,179 @@ SUBSYSTEM_DEF(garbage)
 		msg += "n/a|"
 	else
 		msg += "TGR:[round((totalgcs/(totaldels+totalgcs))*100, 0.01)]%"
+	msg += " P:[pass_counts.Join(",")]"
+	msg += "|F:[fail_counts.Join(",")]"
```

Comment:
Should this string only be formatted in one place? I'm worried about a performance situation like we has with the blackbox

Reply:
yes... I should convert this to a list.

Reply:
But it would out of scope for this pr, as i'd have to convert every subsystem